src/localization/en.json

```diff
@@ -53,5 +53,15 @@
     "published": "Published",
     "statistics": "Statistics",
     "preview": "Preview"
+  },
+  "layouts": {
+    "title": "Select page layout",
+    "questionnaire": "Questionnaire",
```

Comment:
Same comment here as mentioned before, is questionnaire the term we are using or is this just referring to a question?

Reply:
It's referring to the layout type. Maybe it could be just question and we use term "Kysymys" in Finnish. Not very sure about this.